commands/buy.ts

```diff
@@ -16,6 +16,7 @@ function buy (uri: string, command: CommandPrompt): void {
       return
     }
     buyContent(uri, settings.account, password).then(contents => {
+      console.log(contents)
```

Comment:
How about add string in front of `contents` like `console.log('buy contents:' + contents)` to show the type of log data?

Reply:
Yes, good idea. I will do it )